packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -2173,3 +2231,14 @@ export enum ProjectNotificationEvents {
 function isBindableBuildImage(x: unknown): x is IBindableBuildImage {
   return typeof x === 'object' && !!x && !!(x as any).bind;
 }
+
+export function isLambdaComputeType(computeType?: ComputeType): boolean {
+  if (!computeType) return false;
```

Comment:
```suggestion
export function isLambdaComputeType(computeType: ComputeType): boolean {
```
On second thought, let's make the property non-optional.
After all, we want to check only defined values.